documentation/directory.md

```diff
@@ -20,11 +20,16 @@ _Note: **@usernames** in bold denote the primary point of contact for each team.
 | [CoVital](https://www.covital.org/) |[@Daniel Pritchett](https://app.slack.com/team/UVACS1MFF) |
 | [#project-hospitals-in-need](https://helpfulengineering.slack.com/archives/C010UK6GQPR)  | [@peter](https://app.slack.com/team/U010AK510F2)|
 | #project-user-database | [@peter](https://app.slack.com/team/U010AK510F2)|
+| https://hospitalheros.care/ | [@Andrew Cantino](https://app.slack.com/team/U01098E93PE) |
 
 # [Projects](https://github.com/search?q=topic%3Aproject+org%3Ahelpfulengineering&type=Repositories)
 
 <!-- Ideally -->
-Each project keeps its own team directory [inside its main repository](https://github.com/search?q=topic%3Aproject+org%3Ahelpfulengineering&type=Repositories)
+Each project keeps its own team directory [inside its main repository](https://github.com/search?q=topic%3Aproject+org%3Ahelpfulengineering&type=Repositories).
+Known project team members listed below
+
+Reach4Help
+* [@Joseph Ashwin](https://helpfulengineering.slack.com/team/U010ZPFB38V)
```

Comment:
Could we move this entry to the table above?

Reply:
@devhawk ^^

Reply:
Can do but I want to indicate these are team members not project liaisons

Reply:
I added a column for "Other Team Members"